privacypolicy-fr.md

```diff
@@ -0,0 +1,101 @@
+# Politique de confidentialité (v. 2)
+
+## Comment les données des utilisateurs seront-elles traitées ?
+Nous respectons votre vie privée. De ce fait, Babykarte ne collecte aucune donnée utilisateur. Aucun cookie n'est enregistré et aucune statistique d'utilisation n'est générée.
+
+Babykarte n'est pas un fournisseur de cartes ou une base de données mais un portail. Toutes les données saisies par l'utilisateur seront transmises à des tiers qui fourniront les données nécessaires à l'affichage des informations souhaitées. De ce fait, des tiers reçoivent des informations personnelles sur les utilisateurs de Babykarte.
+
+La liste suivante indique quels tiers sont utilisés, à quoi ils servent et où vous pouvez trouver leur politique de confidentialité.
+
+## Thiers utilisés
```

Comment:
```suggestion
## Tiers utilisés
```

Reply:
I have forgotten to do a spell checking on this file, I will fix this issue and other one with a spell checker

Reply:
Done in 7a4bf3a